docs-source/spring/content/development/cli/_index.md

```diff
@@ -1,16 +1,10 @@
 ---
-title: Using the CLI
+title: Using the Oracle Spring CLI
 ---
 
-## Setup
+The Oracle Backend as a Service offer a Spring Cloud command-line tool, `oracle-spring`, to be use. The CLI commands simplify the deployment of applications containing microservices as well as bindings with the resources they use.
```

Comment:
use correct product name. 
offers is plural
"to be use" means nothing - remove it
"they" -> "those applciations" 
"applications containing microservices" -> "microservices applications"